funnel/views/project.py

```diff
@@ -729,32 +731,35 @@ def save(self):
                     'status': 'error',
                     'error': 'project_save_form_invalid',
                     'error_description': _(
-                        "Something went wrong, please reload and try again"
+                        "Something went wrong. Reload and try again"
```

Comment:
We need less ambiguity in error statements. This is a CSRF error, so the message can say it: "This page timed out. Refresh and try again"